docs/graph-algorithms/kruskals-algorithm.mdx

```diff
@@ -0,0 +1,575 @@
+---
+title: Kruskal's Algorithm
+slug: /graph-algorithms/kruskals-algorithm
+---
+## Introduction
+
+Kruskal's Algorithm is a popular algorithm to find out the minimum spanning trees using the greedy approach. This algorithm was described by Joseph Bernard Kruskal, Jr. in 1956. 
+
+Kruskal's algorithm originally places all the vertices of the graph isolated from one another, to create a forest of single node trees, and then merges these trees step by step, combining at every iteration any two of the trees with some edge of the original graph.
+
+## Explanation
+
```

Comment:
Can we have a diagram here? Since we have a strict policy of no external images, you can make them on your own using a prototyping tool.

Reply:
Will add the diagram shortly :)